site/content/blog/eth2-dev-chain.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Setting up private Ethereum 2.0 development chain
+date: 2020-03-26T09:00:00-00:00
+image: /images/blog/ethereum20-validator.png
+draft: false
+keywords:
+- ethereum 2.0
+- proof of staking
+- dev
+- lighthouse
+- chainguardian
+description: Learn how to setup eth2 development chain using lighthouse client. 
```

Comment:
Keeping it short (because of truncate on web, google etc.): `How to setup Eth2 development chain using Lighthouse client`